frontend/src/components/Access.jsx

```diff
@@ -0,0 +1,79 @@
+import { useState } from "react";
+import PropTypes from "prop-types";
+import { NavLink, useNavigate } from "react-router-dom";
+import RegistrationForm from "./RegistrationForm";
+
+export default function Access({ setAccessVisible }) {
+  const navigate = useNavigate();
+  const [isVisible, setIsVisible] = useState(false);
+  const [option, setOption] = useState(false);
+  const handleClick = () => {
+    setAccessVisible(true);
+    localStorage.setItem("accessVisible", true);
+  };
+
+  return (
+    <div className="relative isolate flex items-center gap-x-6 overflow-hidden bg-green px-6 py-16 sm:px-3.5 sm:before:flex-1">
+      {!isVisible ? (
+        <div className="flex flex-wrap items-center gap-x-4 gap-y-2">
+          <p className="text-center leading-6 text-gray-900 text-2xl">
+            <strong className="font-semibold">Eating nam nam 2024</strong>
+            <svg
+              viewBox="0 0 2 2"
+              className="mx-2 inline h-0.5 w-0.5 fill-current"
+              aria-hidden="true"
+            >
+              <circle cx={1} cy={1} r={1} />
+            </svg>
+            Rejoigner nous pour plus de recettes de partage et de bonne humeur.
```

Comment:
Can you fix the french errors please ? "rejoignez" and add "," between "recettes" and "de"